load-generator/README.md

```diff
@@ -32,4 +32,14 @@
 ### StatsD Metrics Load Test Sample Command,
 ```
 ./gradlew :load-generator:run --args="metric -r=100 -u=localhost:8125 -d=statsd"
+```
+
+### Zipkin Trace Load Test Sample Command,
+```
+./gradlew :load-generator:run --args="trace -r=100 -u=http://localhost:9411 -d=zipkin"
```

Comment:
Just curious,  where do we we define those port numbers and the are targeting to the receivers in otel col? I guess they were originally from existing stand alone daemon for ziplin and jaeger?

Reply:
> Can we split the PR (finally, it's my turn to say this) to merge the trace generator logic from https://github.com/Omnition/synthetic-load-generator first, I suppose most of the code are identical? ... I still prefer to move it into load generator folder

They're not identical. I extract and refactor the core code to a generic data emitter client.

Reply:
> Just curious, where do we we define those port numbers and the are targeting to the receivers in otel col? I guess they were originally from existing stand alone daemon for ziplin and jaeger?

yes. it is the default port introduced in Jaeger and Zipkin receiver from upstream. I think they're also from original jaeger/zipkin agents.